extensions/amp-story-shopping/0.1/test/test-amp-story-shopping-config.js

```diff
@@ -20,19 +20,8 @@ describes.realWin(
   (env) => {
     let pageElement;
 
-    const defaultInlineConfig = {
-      items: [
-        {
-          'productId': 'city-pop',
-          'productTitle': 'Plastic Love',
-          'productPrice': 19,
-          'productPriceCurrency': 'JPY',
-        },
-      ],
-    };
-
     const keyedDefaultInlineConfig = {
-      'city-pop': defaultInlineConfig.items[0],
+      'art': remoteConfig.items[0],
```

Comment:
By using remote config as inline, the tests that differentiate between the result from `src` and `<script>` because their objects are the same. 

If the tests are failing without this change, it's a legitimate failure.

Reply:
Good point. Yeah, I had to update the unit tests since the default config has changed and are no longer using city pop.